### PR TITLE
LUN-3070: Add Client Account analytics page guide

### PR DESCRIPTION
**Pull request by rbruggem**
The guide describes how to embed a logistics analytics pages into our
customers' portals for their customers.

### CODE REVIEW

**Comment by rbruggem** [comment]
https://linear.app/lune/issue/LUN-3070/create-documentation-guide

**Comment by rbruggem** [comment]
Here's the page https://client-account-analytics.lune-docs-azp.pages.dev/guides/logistics-customer-analytics

File: docs/guides/logistics-customer-analytics.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+<div className="paragraphSections">
+
+<div>
+
+## Overview
+
+In this guide, you will learn how to embed a link to a no-code analytics page in order for you customers to:
+1. view shipment emissions analytics
+2. explore emission estimates for their shipments
+3. understand how individual estimates were calculated.
```
Comment (raguiar9080) [inline docs/guides/logistics-customer-analytics.md:21]: I think the first letter should be capitalized here.

File: docs/guides/logistics-customer-analytics.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+### Create Client Accounts with handles
+
+In order for your customers to have their own analytics and individual emission estimates pages, you are required to create a Client Account for each customer.
+Client Accounts can be created via [API](/api-reference/client-accounts/create-client-account) or [Logistics CSV spreadsheet](/guides/logistics-sheets#reference).
+
+For Client Account details follow [this link](/key-concepts/client-accounts).
+
+When creating a Client Account provide a `handle`.
+The handle must be unique per organisation and can only contain lowercase alphanumeric characters, underscore and dashes.
```
Comment (raguiar9080) [inline docs/guides/logistics-customer-analytics.md:66]: Should we say it's unique per organisation and account type?

**Review by raguiar9080 — APPROVED** [review]

**Pull request merged**